app-devel/lazygit/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=lazygit
+PKGSEC=devel
+PKGDES="simple terminal UI for git commands"
+PKGDEP="git glibc"
```

Comment:
`glibc` is widely available and we don't normally add it to the Depends section.